src/system-script.js

```diff
@@ -5,7 +5,7 @@ let outerSystem = SystemJS;
 window.define = window.canopyDefine = function(name, deps, m) {
 	if (typeof name === "string") {
 		nameCounter++;
-		const newName = `__script__${nameCounter}`;
+		const newName = `__samds__${name}__${nameCounter}`;
```

Comment:
Put the service name inside the generated name. This makes debugging issues much easier.